src/box/lua/schema.lua

```diff
@@ -2502,6 +2561,23 @@ base_index_mt.alter = function(index, options)
     end
     return box.schema.index.alter(index.space_id, index.id, options)
 end
+base_index_mt.tuple_pos = function(index, tuple)
+    check_index_arg(index, 'tuple_pos')
+    if not tuple then
+        box.error({reason="Usage index:tuple_pos(tuple)"})
+    end
+    local pos = ffi.new('const char *[1]')
+    local pos_end = ffi.new('const char *[1]')
+    local ibuf = cord_ibuf_take()
+    local data, data_end = tuple_encode(ibuf, tuple)
+    local nok = builtin.box_index_tuple_position(index.space_id, index.id, data,
+                                                data_end, pos, pos_end) ~= 0
+    cord_ibuf_put(ibuf)
+    if nok then
+        box.error()
+    end
+    return ffi.string(pos[0], pos_end[0] - pos[0])
```

Comment:
I think you should clean up the region here, as well.

Please add a test that runs `select` and `position` Lua functions in a loop and checks that `box.runtime.info().used` doesn't grow without bounds.

Reply:
Done